### PR TITLE
Fixes #18335 - add pg_dumpall -g to online-backup

### PR DESCRIPTION
**Pull request by jcpunk**
Per redmine ticket, I've added pg_dumpall -g to online-backup

Since I'm looking at katello-backup stuff right now, may as well take care of this one....

### CODE REVIEW

**Review by chris1984 — COMMENTED** [review]
[test]

**Comment by ehelms** [comment]
[test]

File: katello/katello-restore [hunk 1]
```diff
@@ -46,6 +46,7 @@ def restore_databases
     `service postgresql start`
     `runuser - postgres -c "dropdb foreman"`
     `runuser - postgres -c "dropdb candlepin"`
+    `runuser - postgres -c "pg_restore -C -d postgres #{DIR}/pg_globals.dump"`
```
Comment (cfouant) [inline katello/katello-restore]: I am getting this error: `pg_restore: [archiver] input file appears to be a text format dump. Please use psql.` I don't think we can use pg_restore with pg_dumpall.

Comment (jcpunk) [reply katello/katello-restore]: Interesting, looks like my DBAs have slipped a wrapper shell script in on me.  I'll get it fixed up.

File: katello/katello-restore [hunk 2]
```diff
@@ -46,6 +46,7 @@ def restore_databases
     `service postgresql start`
     `runuser - postgres -c "dropdb foreman"`
     `runuser - postgres -c "dropdb candlepin"`
+    `runuser - postgres -c "psql -f #{DIR}/pg_globals.dump postgres"`
```
Comment (cfouant) [inline katello/katello-restore]: I'm getting some errors here that can be ignored. Could you add ` 2>/dev/null` to the end of this command? Thanks @jcpunk !

Comment (jcpunk) [reply katello/katello-restore]: Can do, I'll have it up in a jiffy

**Comment by jcpunk** [comment]
Updated to hide `psql -f #{DIR}/pg_globals.dump postgres` errors

**Comment by cfouant** [comment]
[test]

**Comment by ehelms** [comment]
Will need a rebase

**Comment by cfouant** [comment]
[test]

**Comment by jcpunk** [comment]
rebased

**Comment by chris1984** [comment]
[test]

**Comment by ehelms** [comment]
@cfouant does this need further review?

**Comment by jcpunk** [comment]
rebased to master

**Comment by cfouant** [comment]
[test]

**Comment by cfouant** [comment]
ACK, thanks @jcpunk 

@ehelms ready for merge

**Pull request merged**